pkg/motor/login.go

```diff
@@ -22,12 +21,13 @@ func (mtr *motorNodeImpl) Login(request mt.LoginRequest) (mt.LoginResponse, erro
 	mtr.Address = request.Did
 
 	// fetch vault shards
-	mtr.callback.OnMotorEvent(ct.MotorCallbackMessage_MTR_LOGGED_IN, false)
+	mtr.callback.OnMotorEvent("Fetching shards from vault", false)
```

Comment:
References to the value represented by each string should be made through some kind of strongly typed value / constant